src/pymovements/dataset/dataset.py

```diff
@@ -87,6 +87,7 @@ def load(
             events_dirname: str | None = None,
             preprocessed_dirname: str | None = None,
             extension: str = 'feather',
+            set_trial_columns: bool = False,
```

Comment:
please revert as this is now fixed in #928

Reply:
Reverted it. I hope I didn't miss anything.